answer_files/10/Autounattend.xml

```diff
@@ -2,6 +2,28 @@
 <unattend xmlns="urn:schemas-microsoft-com:unattend">
     <servicing/>
     <settings pass="windowsPE">
+        <component name="Microsoft-Windows-PnpCustomizationsWinPE"
+            publicKeyToken="31bf3856ad364e35" language="neutral"
+            versionScope="nonSxS" processorArchitecture="amd64"
+            xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State">
+
+            <!--
+                 This makes the VirtIO drivers available to Windows, assuming that
+                 the VirtIO driver disk
+                 (https://alt.fedoraproject.org/pub/alt/virtio-win/latest/images/bin/)
```

Comment:
This link seems deprecated as the readme in that folder shows me.

Maybe update it to `https://docs.fedoraproject.org/en-US/quick-docs/creating-windows-virtual-machines-using-virtio-drivers/index.html#virtio-win-direct-downloads` ?

Reply:
Ah, good catch. It looks like I actually used the new repository but kept the link to the old one. Updated.